.env

```diff
@@ -6,25 +6,30 @@ NEXT_PUBLIC_NAT_WEB_SOCKET_DEFAULT_ON=false
 NEXT_PUBLIC_NAT_CHAT_HISTORY_DEFAULT_ON=false
 NEXT_PUBLIC_NAT_RIGHT_MENU_OPEN=false
 NEXT_PUBLIC_NAT_ENABLE_INTERMEDIATE_STEPS=true
+NEXT_PUBLIC_NAT_SHOW_DATA_STREAM_DEFAULT_ON=false
+NEXT_PUBLIC_NAT_ADDITIONAL_VIZ_DEFAULT_ON=false
 
 # =============================================================================
 # Backend Configuration
 # =============================================================================
-# REQUIRED: Backend server address (without protocol)
-# Used for both HTTP API and WebSocket connections
-# Protocols are added automatically: http/ws in dev, https/wss in production
-NEXT_PUBLIC_NAT_BACKEND_ADDRESS=127.0.0.1:8000
+# REQUIRED: Your API server backend address
+# Always include the protocol (http:// or https://)
+NAT_BACKEND_URL=http://127.0.0.1:8000
```

Comment:
MCP uses NEXT_PUBLIC_SERVER_URL. Should we 
1. move that to NAT_BACKEND_URL (or)
2. add NEXT_PUBLIC_SERVER_URL to the .env?
It will be good to have that work out of the box